eZ/Bundle/EzPublishCoreBundle/Resources/views/content_fields.html.twig

```diff
@@ -27,8 +27,8 @@
 {% block ezxmltext_field %}
 {% spaceless %}
     {% set field_value = field.value.xml|xmltext_to_html5 %}
-    {{ block( 'simple_block_field' ) }}
 {% endspaceless %}
+{{ block( 'simple_block_field' ) }}
```

Comment:
Now the spacesless part is doing nothing, so this looks like hack. Did you check why it was spaceless in the first place?